tests/Tactician/CommandBusConsumerTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Burrow\tests\Tactician;
+
+use Burrow\Tactician\CommandBusConsumer;
+use Burrow\Tactician\CommandSerializer;
+use League\Tactician\CommandBus;
+use League\Tactician\Plugins\NamedCommand\NamedCommand;
+
+class CommandBusConsumerTest extends \PHPUnit_Framework_TestCase
+{
+    private $serializer;
+
+    private $commandBus;
+
+    protected function tearDown()
+    {
+        \Mockery::close();
+    }
+
+    protected function setUp()
+    {
+        $this->serializer = \Mockery::mock(CommandSerializer::class);
+        $this->commandBus = \Mockery::mock(CommandBus::class);
+    }
+
+    /**
+     * @test
+     */
+    public function it_should_handle_the_deserialized_command()
+    {
+        $serializedCommand = json_encode([ 'foo' => 'bar' ]);
+        $command = \Mockery::mock(NamedCommand::class);
```

Comment:
True, but as I don't have an accessible class to implement it, mocking it was easier.